src/components/Experiment/QuickOutput/Outputs/ObjectDetection/ObjectDetection.js

```diff
@@ -20,7 +20,7 @@ export default function ObjectDetection(props) {
     if (sections.length === 0) return <NoPredictions modelId={props.trial.model.id}/>
 
     return <div className={getElement("top-row")}>
-      <ObjectDetectionImage img={props.trial.inputs[0]}
+      <ObjectDetectionImage img={props.trial.inputs[0]["src"]}
```

Comment:
I think you can write `props.trial.inputs[0].src` rather than `["src"]`

Reply:
Noted Will do that